ui/raidboss/popup-text.ts

```diff
@@ -260,7 +260,9 @@ class OrderedTriggerList {
   }
 }
 
-const isObject = (x: unknown): x is { [key: string]: unknown } => x instanceof Object;
+const isObject = (x: unknown): x is { [key: string]: unknown } => {
+  return x !== null && x instanceof Object && !Array.isArray(x);
```

Comment:
This makes sense to me but it may be worth a comment as to why this is necessary, something like "javascript considers `[]` to be an object"?

(also is the `null` check here required?)

Reply:
Left a comment, and removed the null check. Good thought! I was thinking of `typeof` and not `instanceof` when I added that.